doc/source/conf.py

```diff
@@ -20,7 +22,7 @@
 
 sys.path.append(os.path.join(root_path))
 project = "PyAEDT"
-copyright = "(c) 2021 ANSYS, Inc. All rights reserved"
+copyright = f"(c) {datetime.datetime.now().year} ANSYS, Inc. All rights reserved"
 author = "Ansys Inc."
 
 # Check for the local config file, otherwise use default desktop configuration
```

Comment:
@akaszynski Few lines below in this file, to get the semantic version, we are doing:
```
with open(os.path.join(root_path, "pyaedt", "version.txt"), "r") as f:
    release = version = f.readline()
```
Do you think that we should expose the version in the `_version.py` with an attribute like this `__version__` and then do in the `conf.py` file:
`release = version = __version__`

Reply:
Both approaches work. Since we already need to import `pyaedt` for the auto documentation to be built, I think it’s easier to:
```py
from pyaedt import __version__
```